docs/docsite/rst/playbooks_loops.rst

```diff
@@ -59,6 +59,32 @@ Also be aware that when combining `when` with `with_items` (or any other loop st
 
 Loops are actually a combination of things `with_` + `lookup()`, so any lookup plugin can be used as a source for a loop, 'items' is lookup.
 
+Additionally, take note that, for convenience, ``with_items`` flattens the
```

Comment:
Please remove "Additionally, take note that, for convenience," and replace with "Please note that"